docs/MO_DG/prepare_model/MO_Python_API.md

```diff
@@ -1,123 +0,0 @@
-# Convert Models Represented as Python Objects {#openvino_docs_MO_DG_Python_API}
-
-@sphinxdirective
-
```

Comment:
why is it removed?

Reply:
The removal of the content above is suggested by @yury-gorbachev,  in CVS--112883.

Reply:
I have moved a part of the removed article to: https://github.com/openvinotoolkit/openvino/pull/18757/files#diff-2b94a53e4a9a9a0190b0f00e88e18bd9ae875eecec0b0395527eb18c1b334f87R152

Reply:
Reverted with the https://github.com/openvinotoolkit/openvino/pull/18757/commits/3bb7d034657abf4bd83da440efdd6d31231d8276 commit